internal/storage/sql/common/segment.go

```diff
@@ -55,7 +55,7 @@ func (s *Store) GetSegment(ctx context.Context, namespaceKey, key string) (*flip
 	segment.CreatedAt = createdAt.Timestamp
 	segment.UpdatedAt = updatedAt.Timestamp
 
-	query := s.builder.Select("id, namespace_key, segment_key, type, property, operator, value, created_at, updated_at").
+	query := s.builder.Select("id, namespace_key, segment_key, type, property, operator, value, created_at, updated_at, description").
```

Comment:
could you swap these around so that `created_at, updated_at` are always the last fields for all our SQL queries?